server/routes/employees.js

```diff
@@ -1,19 +1,35 @@
 var express = require("express");
 var router = express.Router();
 
+const Pool = require("pg").Pool;
+
+const pool = new Pool({
+  user: "postgres",
+  password: "miW478facabbghy78",
+  host: "127.0.0.1",
+  database: "moffitt",
+  port: 5432
+});
+
 router.post("/employees", function(req, res) {
   var firstName = req.body.firstName;
   var lastName = req.body.lastName;
+  var email = req.body.email;
   console.log("firstName", firstName);
   console.log("lastName", lastName);
-  console.log("working");
+  console.log("email", email);
   res.json({ Successful: true });
```

Comment:
The convention for json keys is that they should be camelCase so let's make all of our successful's lowercase

Reply:
done